documentation-website/Writerside/topics/Breaking-Changes.md

```diff
@@ -1,5 +1,11 @@
 # Breaking Changes
 
+## 0.57.0
+* Insert, Upsert, and Replace statements will no longer implicitly send all default values (except for client-side default values) in every SQL request. 
```

Comment:
I'd consider also adding an example, in the event this is vague and potentially scares users. Especially users who maybe didn't realize this was being done and don't understand.
I'd suggest adding a code block with a simple table with both a `default()` and `defaultExpression()` column, then show the insert DSL with the SQL that was generated before and the SQL that will be sent now (H2 is enough).